docs/overview/index.md

```diff
@@ -5,18 +5,40 @@ slug: /
 
 # What is Cradl?
 
-Cradl is the platform where developers can **create custom machine learning models for reading document data**. Use your data to build your own tailor-made models for extracting structured data from semi-structured documents. Define, train and deploy your model on Cradl - **no machine learning expertise required.**
+Cradl is a platform that enables developers to build their own document data capture APIs with deep learning.
 
-![How Cradl works illustration](/img/how-cradl-works.png)
+Our approach to data capture is quite different than most, and is centered around a single design goal:
 
-## Structured data extraction
+:::info Design Philosophy
+Be the easiest way for developers to build _customized data capture APIs_ powered by _deep learning models_.
+:::
 
-Every day, everywhere, everyone parses information from documents such as invoices, receipts, price tags and ID cards. The document formats can vary, the information does not. We want to read amounts, dates, names, VAT details, SWIFT codes, invoice and phone numbers. We _know_ the data exists in that image or that PDF, we _just need to parse it_, but doing so requires a human or AI to read it. This is the hallmark of semi-structured data: it lacks a fixed format, but there's valuable information in it _**that can be structured**_.
+## Who is Cradl for?
 
-Cradl's approach to extracting structured data is _**learning by example, not by instruction**_. No need to customize stencils and keywords for every single document type - our machine learning models teach themselves how to read the data you need from your collection of documents. All you need to do is upload enough example data for our models to get going, tell them what to look for and sit back as your custom machine learning model gets trained. Oh - and you don't have to worry about deploying the model, we take care of that, too.
+Cradl is built for developers, and is used by developers e.g. in _financial institutions_, _public sector_, large newspapers and in ERP software companies. If your company:
+
+* process high volumes of semi-structured documents like invoices, receipts, forms, etc.
+* require high-accuracy data capture
+
+then Cradl is for you.
+
+:::tip Need inspiration?
+Check out some of our [use cases](/use-cases)!
```

Comment:
```suggestion
Check out some of our [use cases](/use-cases)! If you are a developer in the _public sector_, _financial institutions_ or an ERP software company, Cradl might be perfect for you.
```

Reply:
This feels a bit out-of-place to me. I suggest leaving it as is.